react/OrderItems.tsx

```diff
@@ -217,14 +222,37 @@ export const OrderItemsProvider: FC = ({ children }) => {
     [updateQuantity]
   )
 
+  const addItemToCart = async (items: Partial<Item>[]) => {
+    const task = async () => {
+      try {
+        const {
+          data: { addToCart: newOrderForm },
+        } = await addToCart({ variables: { items } })
+
+        return newOrderForm
+      } catch (error) {
+        console.error(error)
+        return null
+      }
+    }
+
+    const newOrderForm = await enqueue(task)
+
+    newOrderForm && setOrderForm(newOrderForm)
+    let error = !newOrderForm
+
+    return { data: newOrderForm, error }
+  }
+
   const value = useMemo(
     () =>
       loading
         ? {
             updateQuantity: noop,
             removeItem: noop,
+            addItemToCart: () => {},
```

Comment:
```suggestion
            addItemToCart: noop,
```

Reply:
`noop` has `void` as return type, and `addItemToCart` returns a promise. What should I do?